src/components/StationSelection.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { Dispatch } from 'react';
+
+const stations = [
+    ['London Euston', 'EUS'],
+    ['Finsbury Park', 'FPK'],
+    ['London Kings Cross', 'KGX'],
+    ['Cambridge', 'CBG'],
+    ['Leeds', 'LDS'],
+];
+
+type StationSelectionProps = {
+    setter: Dispatch<string>;
+}
+
+const handleChange = (event: React.ChangeEvent<HTMLSelectElement>, setter: Dispatch<string>) => {
+    setter(event.target.value);
+};
+
+const StationSelection: React.FC<StationSelectionProps> = ({ setter }) => {
+    const stationHtml = stations.map((station) => {

```

Comment:
I think the React-Select library can make this a little nicer: https://react-select.com/home#welcome